bin/wiki_entity_linking/wikidata_train_entity_linker.py

```diff
@@ -77,6 +77,8 @@ def main(
     if labels_discard:
         labels_discard = [x.strip() for x in labels_discard.split(",")]
         logger.info("Discarding {} NER types: {}".format(len(labels_discard), labels_discard))
+    else:
+        labels_discard = []
 
```

Comment:
This fix is definitely fine to have here, thanks!

I'm actually surprised this throws an error. I'm assuming this happens because of [this line](https://github.com/explosion/spaCy/blob/e38a37e5a6dc1320053508879460defe05c196a6/spacy/pipeline/pipes.pyx#L1301) in `pipes.pyx` ? I was assuming that `self.cfg.get("labels_discard", [])` would return `[]` if the key for `labels_discard` is passed with value `None` in `cfg`, but I guess that's not the case then?

Reply:
Hi @svlandeg . I think that's due to the fact that at line 107 the config the key `labels_discard` got passed in already. Therefore the `self.cfg.get("labels_discard", [])` would return the value for existing key rather than the default. 

I am a big fan of SpaCy's NEL! Thank you for all the nice works!

Reply:
Yep, you're right :-) Could you perhaps also fix the code in `pipes.pyx` itself? In case that gets called from a different script than the `wikidata_train_entity_linker.py` one?

We could rewrite that line to two lines:
```
to_discard = self.cfg.get("labels_discard", [])
if to_discard and ent.label_ in to_discard: 
```

Which would make it works with both `None` as value as well as `[]`. Or something such :-)